radio/src/targets/taranis/STM32_USB-Host-Device_Lib_V2.1.0/Libraries/STM32_USB_Device_Library/Class/msc/src/usbd_msc_scsi.c

```diff
@@ -687,7 +685,7 @@ static int8_t SCSI_ProcessRead (uint8_t lun)
              len);
   
   
-  SCSI_blk_addr   += len; 
+  SCSI_blk_addr   += len / SCSI_blk_size; 

```

Comment:
Pretty sure because:
* len is some number multiplied by SCSI_blk_size 
* MSC_MEDIA_PACKET is multiple of SCSI_blk_size (which is equal to BLOCKSIZE which is 512 in our case)